openapi/listings/listings.yaml

```diff
@@ -261,6 +314,13 @@ components:
                   description: Does the website match the business’s website in their Vendasta Business Profile
                   type: boolean
             accuracyStatusCode:
-              description: Indicates if reporting is available. If available indicates if the listing was found. If the listing was found indicates if all the data is correct on the listing.
+              description: |-
+                Indicates if Vendasta's independent accuracy reporting is available, found and whether or not it is active.
+
+                accuracyStatusUnknown - Default error value, should never be used intentionally
```

Comment:
As discussed in person, please remove the prefixes from the codes for all enums.